ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py

```diff
@@ -148,6 +148,29 @@ def _get_dtype_and_square_matrix(draw):
     return dtype, mat
 
 
+# bmm helper function
+@st.composite
+def _get_dtype_and_values_bmm(draw):
```

Comment:
As the input tensor should be a `3-D` tensor we can simply restrict this in `helpers.dtype_and_values` by specifying the `min_num_dims` and `max_num_dims`  as `3` with no need for extra strategy. With this, we can also control the size of each dimension of the tensor.

Reply:
Thank you so much for the swift feedback.
I initially thought of using the above method to:
dtype_and_x=helpers.dtype_and_values(
    available_dtypes=("float"),
    shared_dtype=True,
    num_arrays=2,
    min_num_dims=3,
    max_num_dims=3,
),

Reply:
This makes sense, you can stick with current approach

Reply:
Please, I want to be clear...is it with the helpers.dtype_and_values above or with the (b, m, k) and (b, k, n) previously implemented?